Documentation/kubernetes/requirements.rst

```diff
@@ -45,6 +45,15 @@ the `Kubernets CNI network-plugins documentation <https://kubernetes.io/docs/con
 Mounted BPF filesystem
 ======================
 
+.. Note::
+
+        Some distributions mount the bpf filesystem automatically. Check if the
+        bpf filesystem is mounted by running the command.
+
+        .. code:: bash
+
+                  mount | grep /sys/fs/bpf
```

Comment:
Is bpf fs guaranteed to be always mounted at `/sys/fs/bpf`? Should we grep e.g. for `type bpf` instead? Or would that be too much of a corner case?

Reply:
I think it is too much of a corner case, and I also think we should make sure that it is mounted at `/sys/fs`